katdal/test/test_datasources.py

```diff
@@ -98,28 +103,45 @@ def make_fake_datasource(telstate, store, cbid, l0_shape, l1_flags_shape=None,
     return view_l0_capture_stream(telstate, cbid, 'sdp_l0') + (l0_data, l1_flags_data)
 
 
+def assert_telstate_data_source_equal(source1, source2):
+    """Assert that two :class:`~katdal.datasources.TelstateDataSource`s are equal."""
+    # Check attributes (sensors left out for now)
```

Comment:
Looks to me like this will iterate over sensors too (although presumably it only checks the most recent value?)

Reply:
Yes, correct on both counts. Because it doesn't fully check the sensors, I thought I'd mention it. (Of course, the unit test has a sensor with a single value which does get checked, but not the associated timestamp.)